### DISCUSSION TITLE
Mi pokedex

### DISCUSSION THREAD

**Discussion by DaveOval** [Launch-X-Latam/MisionFrontEnd#345] [Show and tell]
Que onda banda ayer justo termine la practica de la pagina y ahora les quiero mostrar los inicios de mi pokedex, lo estoy haciendo con puro css y el fondo y el logo de pokedex lo descargue de google

link del repo: https://github.com/DaveOval/Pokedex/settings/pages
link de la pagina: https://daveoval.github.io/Pokedex/

**Comment by doguedogue** [comment]
genial 🚀🚀🚀

**Comment by Cerecero** [comment]
Te quedo genial, solo una cosa en tu post, el link a tu repo esta equivocado deberia ser este https://github.com/DaveOval/Pokedex/ .

**Comment by Aldo-Luis-Medina-Resendiz** [comment]
la neta sí esta chido, solo que no cambia el nombre, solo dice nombre del pokemon
